### DISCUSSION TITLE
Syntax Hightlighting

### DISCUSSION THREAD

**Discussion by TaxBusby** [actions/github-script#385] [Ideas]
This action is really awesome. It would be super next level if there was a VSCode extension that would give proper syntax highlighting to the JS we write in it. It is very helpful to be able to write complex logic in JS instead of bash and syntax highlighting would make our CI/CD code even more maintainable!

**Comment by joshmgross** [comment]
👋 @TaxBusby I agree! Do you mind opening this feature request in https://github.com/github/vscode-github-actions?

**Comment by TaxBusby** [comment]
Sure, opened: https://github.com/github/vscode-github-actions/issues/194

**Comment by joshmgross** [comment]
Closing in favor of https://github.com/github/vscode-github-actions/issues/194
